python-checks/src/test/resources/checks/redundantJump.py

```diff
@@ -250,7 +251,51 @@ def redundant_return_inside_else_block():
         print(e)
     else:
         print(42)
-        return # Noncompliant
+        return # FN
+
+def non_redundant_return_inside_except():
+    try:
```

Comment:
I don't see how any of the new test cases could be considered as having a redundant jump. I believe we need `finally` blocks to have `successor == syntacticSuccessor`.